sensor/kubernetes/clusterhealth/updater.go

```diff
@@ -157,6 +163,44 @@ func (u *updaterImpl) getAdmissionControlInfo() *storage.AdmissionControlHealthI
 	return &result
 }
 
+func (u *updaterImpl) getLocalScannerInfo() *storage.LocalScannerHealthInfo {
```

Comment:
It's just itching to extract a generic function out of this once we have generics :-)